### PR TITLE
Set the missing conditions when creating new process groups

### PR DESCRIPTION
**Pull request by johscheuer**
I thought we got an issue for that.

### CODE REVIEW

File: api/v1beta1/foundationdbcluster_types.go [hunk 1]
```diff
@@ -484,12 +484,18 @@ func (processGroupStatus *ProcessGroupStatus) IsExcluded(remainingMap map[string
...
-		Remove:                 false,
-		Excluded:               false,
-		ProcessGroupConditions: make([]*ProcessGroupCondition, 0),
+		ProcessGroupID: processGroupID,
+		ProcessClass:   processClass,
+		Addresses:      addresses,
+		Remove:         false,
+		Excluded:       false,
+		ProcessGroupConditions: []*ProcessGroupCondition{
+			NewProcessGroupCondition(MissingProcesses),
```
Comment (johscheuer) [inline api/v1beta1/foundationdbcluster_types.go:493]: I also thought about adding a new `Initialized` condition that will be reset the first time it appears in the update status reconciler.

**Comment by brownleej** [comment]
This looks like it's addressing #678. Is that correct?

**Review by brownleej — APPROVED** [review]
👍

**Comment by johscheuer** [comment]
> This looks like it's addressing #678. Is that correct?

Yes, I didn't find it when I searched 🤦

**Pull request merged**